.pipelines/ci.yml

```diff
@@ -570,7 +570,9 @@ stages:
 
       - script: |
           set CUDA_PATH=$(Agent.TempDirectory)\v11.8
-          call .\build.bat -T cuda="%CUDA_PATH%" -DOCOS_ENABLE_CTEST=ON -DOCOS_USE_CUDA=ON -DCMAKE_CUDA_ARCHITECTURES=70;86^
+          call .\build.bat -T cuda="%CUDA_PATH%" -DOCOS_ENABLE_CTEST=ON^
+            -DCMAKE_CUDA_FLAGS_INIT=--allow-unsupported-compiler^
```

Comment:
One dash

Reply:
the initial commit is one dash which failed too, and some examples from GH suggest two dashes, but it doesn't work either. 
According to the log, CMAKE_CUDA_FLAGS_INIT doesn't change the nvcc command line, need to figure that out.